### PR TITLE
improve env usage, install rabbitmq

### PR DESCRIPTION
**Pull request by ahilles107**
| Q                         | A
| ------------------------- | ---
| Bug fix?                  | yes
| New feature?              | yes
| BC breaks?                | no
| Deprecations?             | no
| Tests pass?               | yes
| Changelog update required | yes
| Fixed tickets             | 
| License                   | AGPLv3

### CODE REVIEW

File: src/SWP/Bundle/AnalyticsBundle/Controller/StatisticsController.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+     *
+     * @Method("GET|POST")
+     *
+     * Receive events
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function receiveAction()
+    {
+        return new Response('received');
```
Comment (takeit) [inline src/SWP/Bundle/AnalyticsBundle/Controller/StatisticsController.php]: This is missing the functionality, why do you push this controller at all?

File: app/AppKernel.php [hunk 2]
```diff
@@ -54,6 +54,7 @@ public function registerBundles()
 
             new FOS\ElasticaBundle\FOSElasticaBundle(),
             new SWP\Bundle\ElasticSearchBundle\SWPElasticSearchBundle(),
+            new OldSound\RabbitMqBundle\OldSoundRabbitMqBundle(),
```
Comment (takeit) [inline app/AppKernel.php]: Do we use it somewhere? I don't see any usage of this besides the configuration.

Comment (ahilles107) [reply app/AppKernel.php]: we will in next PR, don't want to remove it now.

File: src/SWP/Bundle/AnalyticsBundle/Controller/StatisticsController.php (hunk 1 above)
Comment (ahilles107) [reply src/SWP/Bundle/AnalyticsBundle/Controller/StatisticsController.php]: i will improve it later this week, don't want to remove it now.

**Pull request merged**